osu.Game/Screens/Edit/Timing/SampleSection.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Graphics.Sprites;
+
+namespace osu.Game.Screens.Edit.Timing
+{
+    internal class SampleSection : Section<SampleControlPoint>
+    {
+        private OsuSpriteText bank;
+        private OsuSpriteText volume;
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Flow.AddRange(new[]
+            {
+                bank = new OsuSpriteText(),
+                volume = new OsuSpriteText(),
+            });
+        }
+
+        protected override SampleControlPoint CreatePoint()
+        {
+            var reference = Beatmap.Value.Beatmap.ControlPointInfo.SamplePointAt(SelectedGroup.Value.Time);
+
+            return new SampleControlPoint
+            {
+                SampleBank = reference.SampleBank,
+                SampleVolume = reference.SampleVolume,
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            ControlPoint.BindValueChanged(point =>
```

Comment:
Not a super big deal for now but I believe this can be refactored better by way of exposing an `OnControlPointChanged()` method from the base class. Reasoning being is because I'm pretty sure this will already be handled incorrectly due to not having `runOnceImmediately = true` here, and every usage would have to know about that.